pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -100,6 +104,6 @@ edk2.mkDerivation projectDscPath (finalAttrs: {
     homepage = "https://github.com/tianocore/tianocore.github.io/wiki/OVMF";
     license = lib.licenses.bsd2;
     inherit (edk2.meta) platforms;
-    maintainers = [ lib.maintainers.raitobezarius ];
+    maintainers = [ lib.maintainers.adamcstephens lib.maintainers.raitobezarius ];
```

Comment:
Please add another commit for the maintenance

Reply:
```suggestion
    maintainers = with lib.maintainers; [ adamcstephens raitobezarius ];
```

Reply:
After formatting, maybe we can bless a linter too :)